netlogo-gui/src/main/lab/gui/RunOptionsDialog.scala

```diff
@@ -95,14 +104,15 @@ class RunOptionsDialog(parent: java.awt.Dialog,
       List(
         Property("spreadsheet", Property.FilePath(spreadsheetFile), I18N.gui("spreadsheet")),
         Property("table", Property.FilePath(tableFile), I18N.gui("table")),
+        Property("stats", Property.FilePath(statsFile), "Stats"),
```

Comment:
The other options here are of the form, "Spreadsheet output", "Table output", etc.  This one is just "Stats".  I'm okay dropping "output" from the other three (to save UI space) or adding it to this one, as long as we're consistent. 

This should also be moved to the UI strings.